### PR TITLE
feat(k6): add missing flags

### LINKED ISSUE #1921: [k6] missing autocomplete for `version`
### Sanity checks

- [X] My issue relates to a specific CLI completion spec (e.g. `git checkout` is missing options in `git` completion spec). If your issue is more general, please create your issue here: [withfig/fig](https://github.com/withfig/fig/issues/new?assignees=&labels=NEED_TO_LABEL&template=main_issue_template.yml)
- [X] I have searched [github.com/withfig/autocomplete/issues](https://github.com/withfig/autocomplete/issues?q=) and there are no duplicates of my issue
- [ ] I would like to work on this.

### What CLI tool does this relate to?

k6

### Which statement makes the most sense?

This completion spec isn't complete

### Issue Details

`k6 version` seems to be missing. I haven't checked others, this just struck my eye.

Probably not the most important command, but maybe still nice to have

### PR DESCRIPTION
**Pull request by oscarsjlh**
Fix #1921 missing version flag, along that version missing, saw three more that were missing and added them.

-  Scale
-  Stats
- Status 
- Version

I didn't make this one from scratch, just followed the pattern that was created before.
Hope everything is correct, any issues, please let me know.

### CODE REVIEW

**Comment by oscarsjlh** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by oscarsjlh** [comment]
recheck

File: src/k6.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import { describe } from "node:test";
+
```
Comment (fedeci) [inline src/k6.ts:2]:
```suggestion
```

File: src/k6.ts [hunk 2]
```diff
@@ -459,6 +461,78 @@ const completionSpec: Fig.Spec = {
         },
       ],
     },
+    {
+      name: "scale",
+      description: "Scale a running test",
+      args: {
+        name: "scale",
+        description: "Scale a running test",
+      },
```
Comment (fedeci) [inline src/k6.ts:470]:
```suggestion
```

File: src/k6.ts [hunk 3]
```diff
@@ -459,6 +461,78 @@ const completionSpec: Fig.Spec = {
...
+        },
+      ],
+    },
+    {
+      name: "stats",
+      description: "Show test metrics",
+      args: {
+        name: "stats",
+        description: "Show test metrics",
+      },
```
Comment (fedeci) [inline src/k6.ts:500]:
```suggestion
```

File: src/k6.ts [hunk 4]
```diff
@@ -459,6 +461,78 @@ const completionSpec: Fig.Spec = {
...
+        },
+      ],
+    },
+    {
+      name: "status",
+      description: "Show test status",
+      args: {
+        name: "status",
+        description: "Show test status",
+      },
```
Comment (fedeci) [inline src/k6.ts:514]:
```suggestion
```

File: src/k6.ts [hunk 5]
```diff
@@ -459,6 +461,78 @@ const completionSpec: Fig.Spec = {
...
+        },
+      ],
+    },
+    {
+      name: "version",
+      description: "Show the application version and exit",
+      args: {
+        name: "version",
+        description: "Show the application version and exit",
+      },
```
Comment (fedeci) [inline src/k6.ts:528]:
```suggestion
```

File: src/k6.ts [hunk 6]
```diff
@@ -459,6 +461,78 @@ const completionSpec: Fig.Spec = {
...
+        description: "Scale a running test",
+      },
+      subcommands: [
+        {
+          name: "-h, --help",
+          description: "Help for scale",
+        },
+        {
+          name: "-m --max",
+          description: "Max avalible virtual users",
```
Comment (fedeci) [inline src/k6.ts:478]:
```suggestion
          description: "Max available virtual users",
```

File: src/k6.ts [hunk 7]
```diff
@@ -459,6 +461,78 @@ const completionSpec: Fig.Spec = {
...
+          args: {
+            name: "max",
+            description: "Max avalible virtual users",
```
Comment (fedeci) [inline src/k6.ts:481]:
```suggestion
            description: "Max available virtual users",
```

**Review by fedeci — CHANGES_REQUESTED** [review]
Thanks! I left some comments to fix before merging the PR!

**Comment by oscarsjlh** [comment]
I saw the changes and saw that you removed:
```

 args: {
        name: "x",
        description: "description",
      },

```
But in that file all the previous commands follow that pattern, is that wrong?, if so, should I remove it from other commands that have that pattern?
```

{
  name: "", 
  description: "",
  args: {
   name: "", 
   description: "",
   },
  subcommnads: [
  "if needed"
],
}
```` 

```

File: src/k6.ts [hunk 8]
```diff
@@ -459,6 +459,62 @@ const completionSpec: Fig.Spec = {
         },
       ],
     },
+    {
+      name: "scale",
+      description: "Scale a running test",
+      subcommands: [
```
Comment (fedeci) [inline src/k6.ts:465]: I missed this one 🤦‍♂️ 
```suggestion
      options: [
```

File: src/k6.ts [hunk 9]
```diff
@@ -459,6 +459,62 @@ const completionSpec: Fig.Spec = {
...
+            name: "vus",
+            description: "Number of virtual users (default 1)",
+          },
+        },
+      ],
+    },
+    {
+      name: "stats",
+      description: "Show test metrics",
+      subcommands: [
```
Comment (fedeci) [inline src/k6.ts:491]:
```suggestion
      options: [
```

File: src/k6.ts [hunk 10]
```diff
@@ -459,6 +459,62 @@ const completionSpec: Fig.Spec = {
...
+        {
+          name: "-h, --help",
+          description: "Help for stats",
+        },
+      ],
+    },
+    {
+      name: "status",
+      description: "Show test status",
+      subcommands: [
```
Comment (fedeci) [inline src/k6.ts:501]:
```suggestion
      options: [
```

File: src/k6.ts [hunk 11]
```diff
@@ -459,6 +459,62 @@ const completionSpec: Fig.Spec = {
...
+        {
+          name: "-h, --help",
+          description: "Help for status",
+        },
+      ],
+    },
+    {
+      name: "version",
+      description: "Show the application version and exit",
+      subcommands: [
```
Comment (fedeci) [inline src/k6.ts:511]:
```suggestion
      options: [
```

**Review by fedeci — CHANGES_REQUESTED** [review]

**Comment by fedeci** [comment]
> I saw the changes and saw that you removed:
> 
> ```
> 
> 
> 
>  args: {
> 
>         name: "x",
> 
>         description: "description",
> 
>       },
> 
> 
> 
> ```
> 
> But in that file all the previous commands follow that pattern, is that wrong?, if so, should I remove it from other commands that have that pattern?
> 
> ```
> 
> 
> 
> {
> 
>   name: "", 
> 
>   description: "",
> 
>   args: {
> 
>    name: "", 
> 
>    description: "",
> 
>    },
> 
>   subcommnads: [
> 
>   "if needed"
> 
> ],
> 
> }
> 
> ```` 
> 
> 
> 
> ```

It is not wrong for a command to have args, but it was wrong for the commands added in this PR to have them. I am not sure if the commands already present should have args or not, but this can be easily checked via the man pages of the k6 cli.

**Review by fedeci — APPROVED** [review]

**Pull request merged**